### PR TITLE
add quiz integration

### PR DESCRIPTION
**Pull request by Drapegnik**
- https://dev.wir.by/be/article/bielaruski/

Screen Shot 2020-07-30 at 2 43 18 AM
Screen Shot 2020-07-30 at 2 43 35 AM

### CODE REVIEW

**Comment by Drapegnik** [comment]
[integrate with ex.co for tests](https://trello.com/c/R1dSP8Sp/513-integrate-with-exco-for-tests)

File: components/specials/Quiz.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  if (d.getElementById(id)) {
+    return;
+  }
+  const js = d.createElement(s);
+  js.id = id;
+  js.src = 'https://embed.ex.co/sdk.js';
+  fjs.parentNode.insertBefore(js, fjs);
+}
+
+const Quiz = ({ id }) => {
```
Comment (UladBohdan) [inline components/specials/Quiz.js]: How about `ExCoQuiz`?

File: components/specials/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.quiz {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  &__icon {
+    color: $primary;
+
+    &--loading {
+      animation: spin 1s linear infinite;
```
Comment (UladBohdan) [inline components/specials/index.module.scss]: Don't we have loading icon in our kit?

File: components/specials/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+@import 'styles/variables';
```
Comment (UladBohdan) [inline components/specials/index.module.scss]: We tend to name files as `quiz.module.scss`, synced with js naming

**Review by UladBohdan — DISMISSED** [review]

**Review by UladBohdan — APPROVED** [review]

**Pull request merged**